### ISSUE TITLE
Connection to remote SSH not working, python extension not loading

### ISSUE THREAD

**Issue by prasiyer** [microsoft/vscode#93263]
What: Connection to remote SSH vscode server not working, python extension also not loading

![image](https://user-images.githubusercontent.com/57724404/77354423-d04f8c80-6d10-11ea-8487-6771ef00d06c.png)
version: 1.43.1

Error message in console: 
(node:7728) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.

I see similar issues that have been reported. But i am not able to see the fix.

Please let me know if you additional information.

**Comment by david-it** [comment]
I had the same problem this morning.

The password prompt started to loop, asking for the password over and over.
Out of curiosity, the same problem appeared both in vscode and vscode-insiders.

One of the errors I was getting looked like this: 
```
[10:21:56.330] Showing password prompt
[10:25:19.361] Got password response
[10:25:19.362] "install" wrote data to terminal: "**********"
[10:25:19.372] > 
> 
[10:25:19.382] > Connection closed by 192.168.1.61 port 22
> The process tried to write to a nonexistent pipe.
> 
[10:25:19.653] "install" terminal command done
[10:25:19.653] Install terminal quit with output: The process tried to write to a nonexistent pipe.
[10:25:19.653] Received install output: The process tried to write to a nonexistent pipe.
[10:25:19.653] Resolver error: The process tried to write to a nonexistent pipe
```

In my case I solved removing completely the folder `.vscode-server` on the server (Ubuntu server 18.04).

**For vscode**:
`sudo rm -R .vscode-server`

**For vscode-insiders**:
`sudo rm -R .vscode-server-insiders`

**Comment by prasiyer** [comment]
David-it - Thanks for sharing your comment. For me problem started with the password prompt starting to loop. I did not have to remove the .vscode-server. Problem resolved itself. However, I lost half day. 
Were you able to find the root cause?

**Comment by david-it** [comment]
> David-it - Thanks for sharing your comment. For me problem started with the password prompt starting to loop. I did not have to remove the .vscode-server. Problem resolved itself. However, I lost half day.
> Were you able to find the root cause?

No, I have no idea about the cause. I just noticed that the problem showed up after the updates of yesterday morning of both vscode and vscode-insider 

**vscode**
Version: 1.43.2 (user setup)
Commit: 0ba0ca52957102ca3527cf479571617f0de6ed50
Date: 2020-03-24T07:38:38.248Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**vscode-insiders**
Version: 1.44.0-insider (user setup)
Commit: 073a24de05773f2261f89172987002dc0ae2f1cd
Date: 2020-03-24T05:37:12.592Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
